diff --git a/index.ts b/index.ts
index d6deaed4c730a24aa5ef7838561c934638c5ef83..9450d36ff5c299b3ede1599e6a2559fb19e884a4 100644
--- a/index.ts
+++ b/index.ts
@@ -57,7 +57,7 @@ const starlightLinksValidatorOptionsSchema = z
      */
     exclude: z.array(z.string()).default([]),
   })
-  .default({})
+  .prefault({})
 
 export default function starlightLinksValidatorPlugin(
   userOptions?: StarlightLinksValidatorUserOptions,
